src/benchmarklib/table_generator.cpp

```diff
@@ -24,50 +24,70 @@
 
 namespace opossum {
 
-std::shared_ptr<Table> TableGenerator::generate_table(const ChunkID chunk_size,
+std::shared_ptr<Table> TableGenerator::generate_table(const ChunkOffset chunk_size,
```

Comment:
Why do we have two `generate_table` methods?

Reply:
this was for the presentation, reverted it